doc-site/src/components/EditDocsButton.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {EzTooltip, EzLink} from '@ezcater/recipe';
+
+const EditDocsButton = ({path}) => {
+  if (!path || !path.fileAbsolutePath) {
+    return null;
+  }
+
+  const gitPath = path.fileAbsolutePath.split('/recipe/')[1];
+
+  const gitLink = `https://github.com/ezcater/recipe/blob/main/${gitPath}`;
```

Comment:
Thoughts on making this the edit page directly?

```suggestion
  const gitLink = `https://github.com/ezcater/recipe/edit/main/${gitPath}`;
```


| Blob   |      Edit      | 
|----------|:-------------:|
| ![image](https://user-images.githubusercontent.com/109814/128741399-25225932-4a16-4f7e-9ea1-d422c651b1ef.png) |  ![image](https://user-images.githubusercontent.com/109814/128741433-9b794141-df2d-4810-8f56-3d20a3d45177.png) |